### ISSUE TITLE
[Feature] Generating one.nr short-links for one.newrelic.com URLs

### ISSUE THREAD

**Issue by skyzyx** [newrelic/newrelic-client-go#634]
### Feature Description

Building on #632: The New Relic One UI has a feature where you can generate a short-link. I would love to see support in this client library for generating the short URL that I can give to my users, which will link them into the New Relic One UI.

I've reverse-engineered the implementation to this:

```bash
curl -XPOST \
    https://urly.service.newrelic.com/ \
    -H "Content-Type: application/json" \
    -H "Accept: application/json" \
    -H "Origin: https://one.newrelic.com" \
    -H "Cookie: login_service_login_newrelic_com_tokens={encoded-json}"
    -d '{"url":"https://one.newrelic.com/xxxxxxx"}' \
```

Skipping any of these will result in either a 401 (expected) or 500 (unexpected) error. The only way I have been able to figure out how to authenticate is by using the login cookie from my browser (i.e., `{encoded-json}`) which contains a token that I can't generate from my end.

If there's another way to authenticate that doesn't involve a web browser, that would greatly simplify the implementation.

### Describe Alternatives

I can pass the links to a third-party link shortener.

**Comment by ctrombley** [comment]
Hey @skyzyx, thanks for the feature request.  This one seems to belong in the CLI as well.   

The CLI uses a different authentication mechanism than the cookie-based one you've discovered here, which is only enabled for the New Relic UI.  The NerdGraph API is our designated integration point for automation use cases such as this, so I think there is some work to be done on the platform to surface this particular API endpoint in NerdGraph before we can build around it.  I will talk with some other internal teams here and find out if this is already on a roadmap.

**Comment by ctrombley** [comment]
Duplicate of newrelic/newrelic-cli#635
